config.go

```diff
@@ -5,7 +5,7 @@ package specs
 // bundle is packaged for distribution.
 type Spec struct {
 	// Version is the version of the specification that is supported.
-	Version string `json:"version"`
+	Version string `json:"spec_version"`
```

Comment:
I'd [rather have a more specific name][1].  Presumably the other versioned things were distinguishing from also have specs (and the current JSON pattern is camelCase, not underscores).

[1]: https://github.com/opencontainers/specs/issues/110#issuecomment-171470005